src/modules/submitScore.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-cycle */
```

Comment:
- [ ] Please do not disable linters. Linters enforce unifrom and good coding standards in a team. 👍🏾